app/main.py

```diff
@@ -1,3 +1,53 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        return Distance(
+            self.km + self.return_number(other)
+        )
+
+    def __iadd__(self, other: int | float | Distance) -> Distance:
+        self.km += self.return_number(other)
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        self.km *= other
```

Comment:
How do you think is it a good idea to change the input object? `self.km *= other` will change the object itself. 
Also, `other` might be Distance, not only int or float.